tracer/src/Datadog.Trace.Trimming/Datadog.Trace.Trimming.nuspec

```diff
@@ -19,6 +19,8 @@
     <tags>Datadog APM tracing profiling instrumentation trimming</tags>
     <repository type="git" url="https://github.com/DataDog/dd-trace-dotnet.git" />
     <dependencies>
```

Comment:
We probably don't need the `<dependencies>` element at all, since we have no dependencies (they are all empty).